tests/functional/n/no/no_self_argument.py

```diff
@@ -16,3 +23,18 @@ def abdc(yoo):  # [no-self-argument]
     def edf(self):
         """just another method"""
         print('yapudju in', self)
+
+    @staticmethod
+    def say_hello():
+        """A standard staticmethod"""
+        print("hello!")
+
+    @MYSTATICMETHOD
+    def say_goodbye():
+        """A staticmethod but using a different name"""
+        print("goodbye!")
+
+    @returns_staticmethod
+    def sum_strings(string1, string2):
```

Comment:
```suggestion
    def concatenate_strings(string1, string2):
```

Reply:
Just noting for other reviewers that this test case already passed. (It's the MYSTATICMETHOD that newly passes.)